netlify/netlify-plugin-nextjs
# fix: correctly cache when using `next export`

### Summary

Currently we look for the cache inside the publish directory. Normally this is fine, because the publish directory is `.next`, however when using `next export` it has a different publish directory. This PR attempts to find the cache directory elsewhere if the site is using next export.

### Test plan

1. Observe the build of the next export demo site. 
2. Check that it finds the cache

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Fixes #1192

![two capybaras](https://pbs.twimg.com/media/FLu_M1ZXwAIKV2A?format=jpg&name=900x900)

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
